content/en/docs/Concepts/crds/clusterserviceversion.md

```diff
@@ -92,15 +92,18 @@ spec:
   customresourcedefinitions:
     owned:
     # a list of CRDs that this operator owns
-    # name is the metadata.name of the CRD
-    - name: cache.example.com
+    # name is the metadata.name of the CRD (which is of the form <plural>.<group>)
+    # version is the spec.versions[].name value defined in the CRD
+    # kind is the CamelCased singular value defined in spec.names.kind of the CRD.
```

Comment:
the comment on `kind` is probably more suitable in line 101, right above `version`

Reply:
fixed